src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -453,10 +453,12 @@ export const validateDisabilityName = (
     err.addError('This needs to be less than 256 characters');
   }
 
-  if (
+  const missingCondition =
```

Comment:
Good point! I don't know that `fieldData` will always be a string, so I'd rather be explicit about it. If `fieldData` is `null` or `undefined` it will error with `trim()`

Reply:
Switched it up to the optional chaining and now I'm failling tests but can't repro the fail locally. ugh i'll look at this again tomorrow 😢  
<img width="637" alt="Screenshot 2024-11-05 at 9 57 45 PM" src="https://github.com/user-attachments/assets/8979bf6b-a79d-4873-8e74-525f066814ca">